pkg/analytics/track.go

```diff
@@ -336,8 +336,5 @@ func track(event string, success bool, props map[string]interface{}) {
 }
 
 func disabledByOktetoAdmin() bool {
-	if okteto.IsOktetoCloud() {
```

Comment:
this can be removed because okteto own clusters have and should be disabled using the backend settings